ci/cron/daily-compat.yml

```diff
@@ -379,6 +376,7 @@ jobs:
         else
             if [ "main" = "$(Build.SourceBranchName)" ]; then
                 git add canton_dep.bzl $bad_file build.sh build.ps1
+                [ -f arbitrary_canton_sha ] && git rm -f arbitrary_canton_sha
```

Comment:
I think we need to set the local field to false inside canton_dep.bzl.

Reply:
Indeed. This is done as part of the `cat` command above, when the "new" `canton_dep` is generated.